root/edit/notes.tt

```diff
@@ -23,6 +23,11 @@
                   <textarea class="edit-note" rows="5" placeholder="[% l('Add an edit note') %]"
                       name="enter-vote.vote.[% index %].edit_note"></textarea>
               </div>
+            <p>
+                  [% l('HTML is not supported, but {url|some basic syntax} is available. Please do always keep the {coc|Code of Conduct} in mind when writing edit notes!',
+                     { coc => doc_link('Code_of_Conduct'), url => doc_link('Edit_Note') }) %]
```

Comment:
Please use consistent keys, e.g.
* `doc_syntax` instead of `url`
* `doc_coc` instead of `coc`

There already are some `doc_*` keys around. It is probably more readable to translators.

Reply:
Good point! :)